src/lib/alloc.c

```diff
@@ -613,9 +613,7 @@ void init_heap(struct sof *sof)
 	struct mm_heap *heap;
 	struct block_map *next_map;
 	struct block_map *current_map;
-	int i;
-	int j;
-	int k;
+	int i, j;
```

Comment:
please follow coding style of one variable per line.

Reply:
> please follow coding style of one variable per line.

Thanks for point it out. I didn't realize we have such coding style in sof project.